sdk/core/perf/CMakeLists.txt

```diff
@@ -51,7 +51,7 @@ endif()
 
 # make sure that users can consume the project as a library.
 add_library (Azure::Perf ALIAS azure-perf)
-target_link_libraries(azure-perf PUBLIC azure-core)
+target_link_libraries(azure-perf PUBLIC azure-core azure-identity)
```

Comment:
core depends on identity now  ?

Reply:
*perf* depends on identity. Perf is not a package.

Core depends on nothing.